paimon-web-server/src/main/resources/i18n/messages_en_US.properties

```
@@ -51,3 +51,4 @@ table.drop.error=Exception calling Paimon Catalog API to drop a table.
 table.rename.error=Exception calling Paimon Catalog API to rename a table.
 cdc.job.exist.error=Paimon CDC job exists.
 cdc.job.not.exist.error=Paimon CDC job is not exist.
+cluster.name.exist=The cluster name {0} already exists.
```

```suggestion
cluster.name.exist=This cluster name {0} already exists.
```
To aligin the naming style of other name exist.